verilog/formatting/formatter_test.cc

```diff
@@ -7257,6 +7257,85 @@ static constexpr FormatterTestCase kFormatterTestCases[] = {
      "function int f;\n"
      "  return $x;\n"
      "endfunction\n"},
+    // String initializers
+    {"string a[] = {\n\"a\"\n};\n", "string a[] = {\"a\"};\n"},
+    {"string abc[] = {\n\"a\",\n\"b\",\n\"c\"\n};\n",
+     "string abc[] = {\"a\", \"b\", \"c\"};\n"},
+    {"string abc[] = {\n"
+     "\"a\",//\n"
+     "\"b\", \"c\"\n"
+     "};\n",
+     "string abc[] = {\"a\",  //\n"
+     "                \"b\",\n"
+     "                \"c\"};\n"},
+    {"string abc[] = {//\n"
+     "\"a\", \"b\", \"c\""
+     "};\n",
+     "string abc[] = {  //\n"
```

Comment:
I wonder if some of these strings would be a bit more readable using c++11 raw string literals to have the expected string shine through more with less escape-backslashes needed.
In general, the strings are fine, leaving it up to you to experiment with that option.

Reply:
Raw literals could be used only for strings that don't have `\n` - otherwise we'll need an extra `"\n"` which are put in separate lines by autoformatter. I think it's better to avoid those few "different" lines and keep everything uniform.
Or did you mean multiline raw strings? Like:
```
    {R"(string abc[]= {//
"a", "b", "c"
};
)",
     R"(string  abc[] = {  //
  "a",
  "b",
  "c"
};
)"}
```